baybe/scaler.py

```diff
@@ -3,15 +3,16 @@
 from __future__ import annotations
 
 from abc import ABC, abstractmethod
-from typing import Callable, Dict, Tuple, Type
+from typing import TYPE_CHECKING, Callable, Dict, Tuple, Type
 
 import pandas as pd
-import torch
-from torch import Tensor
 
 from baybe.utils.dataframe import to_tensor
 
-_ScaleFun = Callable[[Tensor], Tensor]
+if TYPE_CHECKING:
+    from torch import Tensor
+
+_ScaleFun = Callable[["Tensor"], "Tensor"]
```

Comment:
If you add an ident, this will be part of TYPE_CHECKING and then no strings are needed

```suggestion
    _ScaleFun = Callable[[Tensor], Tensor]
```